bin/lint

```diff
@@ -0,0 +1 @@
+go list ./... | xargs golint
```

Comment:
`go run vendor/github.com/golang/lint/lint.go`
1. Would it be more efficient to call `lint` only once, instead per-package
2. Seems like we check style on `vendor` also, see how `bin/test` is made

Reply:
3. Would be nice to actually use installed dependency in helper `bin/lint`
e.g `go run vendor/github.com/golang/lint/lint.go`
What google says about best practices about that?

Reply:
>3. Would be nice to actually use installed dependency in helper bin/lint
e.g go run vendor/github.com/golang/lint/golint/*.go nat

Good point, updated.